ci/scripts/unit.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -eux
 
-export GOPATH=$(pwd)/go
+export GOPATH=$(pwd)
```

Comment:
Maybe we can rename this var? (as it is no longer GOPATH because we are using go modules),
I was doing the following in other repos:
```
cwd=$(pwd)
pushd $cwd/florence
...
```